api/v1/db/oracledb/games-dao.js

```diff
@@ -84,4 +85,31 @@ const getGameById = async (id) => {
   }
 };
 
-module.exports = { getGames, getGameById };
+/**
+ * @summary Inserts row into the game table
+ */
+const postGame = async (body) => {
+  const connection = await conn.getConnection();
+
+  try {
+    // Bind newly inserted game row ID to outId
+    // We can use outId to query the newly created row and return it
+    const { attributes } = body.data;
+    attributes.outId = { type: oracledb.NUMBER, dir: oracledb.BIND_OUT };
+    const sqlQuery = `
+      INSERT INTO VIDEO_GAMES (NAME, RELEASE_DATE, DEVELOPER_ID)
+      VALUES (:name, TO_DATE(:releaseDate, 'YYYY/MM/DD'), :developerId)
```

Comment:
Based on your openapi file, the `releaseDate` field should follow [month/day/year](https://github.com/osu-mist/game-curator-api/blob/master/openapi.yaml#L508) format, however, I got the following error when sending request with the `releaseDate: "1/1/2019"`:

```json
{
    "errors": [
        {
            "status": "400",
            "code": "1400",
            "title": "Bad Request",
            "detail": "Error in path: 'data.attributes.releaseDate', location: 'body', message: 'should match format \"date\"'",
            "links": {
                "about": "https://developer.oregonstate.edu/documentation/error-reference#1400"
            }
        }
    ]
}
```

Base on the [OpenAPI definition](https://swagger.io/docs/specification/data-models/data-types/), if you specified `date` in data type, the field should follow the definition in RFC 3339, section 5.6, for example, 2017-07-21.

Reply:
Also, if I try to send this body request, I got a 500:

```json
{
	"data": {
		"type": "game",
		"attributes": {
			"name": "test",
			"releaseDate": "2019-01-01",
			"developerId": "ABC"	
		}
	}
}
```

Can you look into this as well?

Reply:
Fixed 500 error by checking if `developerId` is NaN

Reply:
Made openapi date example and game-serializer output consistent with RFC 3339, section 5.6.